dependenciesCopy.js

```diff
@@ -0,0 +1,33 @@
+const fs = require('fs');
```

Comment:
I think it would be great if we could just move this down into the template folder. If we need to for testing we can have a whole other repo or an examples folder.

Reply:
Should we move all the files in the template folder or just this one?
We can copy all the files, still make it easy to work in this repo itself but that would need some change in the scripts.
If we continue with the current setup, I think npmignore can help us and we can ignore all the files in root directory except template.json and template (which are the only things required by CRA).
Interesting idea of making another repo for development, but won't it make working with the project difficult because, for every change, the developer would need to change code in two repositories. 
I will write the documentation explaining the structure of code which can make it easy to get started with.